tripal/src/TripalField/TripalFieldItemBase.php

```diff
@@ -214,7 +225,8 @@ public function fieldSettingsForm(array $form, FormStateInterface $form_state) {
                 ['@term' => $termIdSpace . ':' . $termAccession]));
             $is_open = TRUE;
           }
-          $default_vocabulary_term = !$default_vocabulary_term ? $term->getName()  . ' (' . $term->getIdSpace() . ':' . $term->getAccession() . ')' : '';
+          // @to-do - logic might be wrong here and would reset default_vocabulary_term?
+          $default_vocabulary_term = !$default_vocabulary_term ? ($term->getName() . ' (' . $term->getIdSpace() . ':' . $term->getAccession() . ')') : '';
```

Comment:
Hmmm. yah. If the default_vocabulary_term is set then this line would set it to an empty string 🤔 Maybe we should do this instead:
```suggestion
          $default_vocabulary_term = !$default_vocabulary_term ? ($term->getName() . ' (' . $term->getIdSpace() . ':' . $term->getAccession() . ')') : $default_vocabulary_term;
```